### DISCUSSION TITLE
Test-pattern

### DISCUSSION THREAD

**Discussion by vitasam** [markadev/AppleII-VGA#7] [Show and tell]
Hello,
I just assembled an "MPV": prototype board + Pico + resistive DAC + VGA connector and got a test-pattern working. The synchronization on my old display is super-stable, a picture is crisp.

Just to clarify: are colors correct (R,G,B channels are correct)? 
![testpattern_display_native](https://user-images.githubusercontent.com/5597505/193125160-72ad5366-1a3f-4c0c-aea2-f8c174347fef.jpg)

**Comment by markadev** [comment]
Cool, glad to hear it works for someone else!  That looks like the right RGB mapping to me.  I'm not sure if it's just the picture, but the blue channel's range looks more limited than the red & green channels.  It's hard to tell from here though... even on the old LCD I used the colors would shift depending on the viewing angle 😄

**Reply by vitasam** [reply]
Hi,

Yes, color tint is expected with my old display too, also it was a question of automatic white balance of my phone's camera- the image is "too warm".

But the main point - colors are not swapped (Red where it should be red, etc.), right?

I captured a new photo - less warm.
![20220930_080306_copy_1526x1110.jpg](https://user-images.githubusercontent.com/5597505/193194611-ec1de4a9-22bf-48f6-8493-eae6cc4c5e46.jpg)

**Reply by vitasam** [reply]
![20220929_222615_copy_750x1000.jpg](https://user-images.githubusercontent.com/5597505/193199724-176c8764-8427-4f3c-b308-043fc96dd5c2.jpg)

The MVP

**Reply by markadev** [reply]
Ah, gotcha.  Yeah you got the colors correct.  The R, G, & B are all in the right place.
